cms/migrations/0015_add_banner_image.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.2.10 on 2020-06-04 12:30
```

Comment:
NIT: better to rename that file with `0015_add_banner_image_in_programdescriptionpage` or so, so that it could be understood better and quicker by just viewing the file name. As of now, it does not speak for where we are adding that field.